popup.js

```diff
@@ -64,7 +64,7 @@ const modalData = [
     },
     image: './images/snap3.png',
     description:
-      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
     tech: ['HTML', 'CSS', 'Javascript', 'github', 'ruby', 'Bootstraps'],
     button: {
       link: 'https://example.com/',
```

Comment:
- [ ] Good job adding the pop-up, in your web page :clap: , However I noticed that as the requirements said that you should stick as much as possible to the Figma design , the pop-up should be above the main section :+1: , I noticed for you that the pop-up is under the navbar, Kindly fix in your mobile version :+1:  

| Figma design | Your design |
| :---: | :---: |
| ![Screenshot from 2023-02-04 02-03-16](https://user-images.githubusercontent.com/80458199/216734195-df79bcc0-9b49-4df5-ad65-d489c24b44f0.png) | ![Screenshot from 2023-02-04 02-20-14](https://user-images.githubusercontent.com/80458199/216734235-8bc153a3-7aa1-435a-8a95-229fa0ed7cd2.png) |